.github/workflows/release-ghcr.yml

```diff
@@ -34,6 +37,13 @@ jobs:
         run: |          
           docker buildx create --use
           docker buildx build --platform linux/amd64,linux/arm64,linux/arm/v7 -t ${{ steps.prepare.outputs.ref }} --push .
+      - name: install cosign
+        uses: sigstore/cosign-installer@48866aa521d8bf870604709cd43ec2f602d03ff2
+        with:
+          cosign-release: 'v1.9.0'
+      - name: sign image
+        run: |
+          COSIGN_EXPERIMENTAL=1 cosign sign -y -f ${{ steps.prepare.outputs.ref }}
```

Comment:
Since this is a multiarch image, is there a desire to sign each individual image? (`cosign sign -r`)